### PR TITLE
change the button input layout to match 3ds' better layout

### PR DESCRIPTION
**Pull request by kane159**

### CODE REVIEW

**Comment by carstene1ns** [comment]
jenkins: ok to test.

File: src/platform/psvita/psp2_ui.cpp [hunk 1]
```diff
@@ -209,7 +209,7 @@ void Psp2Ui::ProcessEvents() {
 	sceCtrlPeekBufferPositive(0, &input, 1);
 	keys[Input::Keys::Z] = (input.buttons & SCE_CTRL_CROSS);
 	keys[Input::Keys::X] = (input.buttons & SCE_CTRL_CIRCLE);
-	keys[Input::Keys::N2] = (input.buttons & SCE_CTRL_TRIANGLE);
+	keys[Input::Keys::F] = (input.buttons & SCE_CTRL_TRIANGLE);
```
Comment (carstene1ns) [inline src/platform/psvita/psp2_ui.cpp:212]: The decision was taken to support yume nikki here.
The 3DS has the advantage to support the "2" key through touch screen. The vita port might need some love.

**Comment by carstene1ns** [comment]
I think for users it is better to have the fast forward than the fps counter, so we should change <kbd>F2</kbd>-><kbd>F</kbd> in line 220 instead.
https://github.com/EasyRPG/Player/blob/77fee9ca8f4bbadf0f3e72d2f0739e7f796a68e1/src/platform/psvita/psp2_ui.cpp#L220

**Comment by Ghabry** [comment]
best way is probably to port the on-screen numpad we have on the Nintendo Switch over to PsVita. Then there are lots of free keys.
Currently the touch screen is only used by the Shader selector.

Will not work on PS TV though.

**Comment by carstene1ns** [comment]
Okay, for PSTV I think some kind of circle menu by button combo like we talked about makes sense, will see if i come up with something.

**Review by carstene1ns — APPROVED** [review]

**Comment by Ghabry** [comment]
with your other PR the mapping of the right analog stick becomes unnecessary. Replace this with movement?

**Comment by carstene1ns** [comment]
We already have movement with left stick and dpad, I have intentionally left it unused for the mentioned "special button stuff"

**Comment by carstene1ns** [comment]
@kane159: Are you around to test a build with touchscreen input?

**Pull request merged**